xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -338,8 +347,25 @@ ol.tagCloud .ultraPopular {
 ##
 $xwiki.ssx.use('XWiki.TagCloud')##
 ##
-## Get tag count map. Key: tag, value: number of occurrences.
-#set ($tagCount = $xwiki.tag.getTagCount("$!tagCloudSpace"))
+##if (!("$!tagCloudSpace" =="") &amp;&amp; !("$!tagCloudSpaces" ==""))
+#if( $tagCloudSpace &amp;&amp; $tagCloudSpaces )
+{{html}}
```

Comment:
I did it but don't know if it is like you expect (I use error and translation)...

(by torturing me with Maven, Vincent didn't like me) ;-)
I launch mvn -e xar:format with pom.xml provided by github repository:
/xwiki-platform/xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/pom.xml
Is it OK?